app/src/components/RegistrationForm.vue

```diff
@@ -0,0 +1,92 @@
+<style lang="scss">
+
+/**
+ *--------------------------------------------------------------------------
+ *	Vue-component for registration-form
+ *--------------------------------------------------------------------------
+ *
+ * 	This is the Vue-component for the registration-form.
+ *
+ * 	This file includes the HTML template, sass-stylesheet 
+ * 	and javascript for the registration-form.
+ *
+ */
+
+.registration-form{
+	
+}
+
+</style>
```

Comment:
Bitte aus Konsistenzgründen die Reihenfolge in allen Komponenten (möglichst) gleich einhalten. Normalerweise:
template, script, style